### DISCUSSION TITLE
Swap any amount of assets on Kroma using iziswap

### DISCUSSION THREAD

**Discussion by ericmanoptr** [blockscout/blockscout#9206] [Support]
I did the task and I don't receive approval from it.

0x02F55D53DcE23B4AA962CC68b0f685f26143Bdb2
0xe8167F1540724f28D3A4305F22599a1f753A5221

0xac9650d800000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000001a0000000000000000000000000000000000000000000000000000000000000012475ceafe6000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000a0000000000000000000000000e8167f1540724f28d3a4305f22599a1f753a5221000000000000000000000000000000000000000000000000000110d9316ec00000000000000000000000000000000000000000000000000000000000000b1e160000000000000000000000000000000000000000000000000000000065abb82e000000000000000000000000000000000000000000000000000000000000002b4200000000000000000000000000000000000001000bb80257e4d92c00c9efcca1d641b224d7d09cfa452200000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000412210e8a00000000000000000000000000000000000000000000000000000000

**Comment by ekawibisono** [comment]
hello @ericmanoptr what is that ?, if you have problem with kroma, you can contact kroma team, this blockscout discord
